study-builder/studies/prion/substitutions.conf

```diff
@@ -2,15 +2,16 @@
     "i18n": {
         "en": {
             include required("i18n/en.conf")
-        },
-        "es": {
-            include required("i18n/es.conf")
-        },
-        "he": {
-            include required("i18n/he.conf")
-        },
-        "zh": {
-            include required("i18n/zh.conf")
         }
+        #,
+        #"es": {
+            #include required("i18n/es.conf")
+        #},
+        #"he": {
+            #include required("i18n/he.conf")
+        #},
+        #"zh": {
+            #include required("i18n/zh.conf")
+        #}
```

Comment:
I think it might be better to leave the language items in the conf files and leave the substitutions here. Just remove language from the study's list of languages in `study.conf`.

Reply:
I tried that and for some reason I was still seeing the language selector.  I think that's a bug in my code for the language selector.  I'll check one more time to make sure

Reply:
List of languages might be cached in your browser. Check again with cache disabled.

Reply:
Actually, this is what happens: I get a compile error: Exception in thread "main" com.typesafe.config.ConfigException$UnresolvedSubstitution: /Users/cdye/IdeaProjects/ddp-study-server/study-builder/studies/prion/consent.conf: 35: Could not resolve substitution to a value: ${i18n.es.consentTitle}

Reply:
Although actually that might be from commenting out stuff in substitutions.conf

Reply:
Figured it out